components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/css/console.css

```diff
@@ -95,23 +99,32 @@
 }
 
 .debuggerToolbar {
-    line-height: 20px;
+    line-height: 12px;
     padding: 5px 10px;
     background: #1d1f21;
     margin: -10px -10px 0;
 }
 
+.consoleToolbar {
+    background: #1d1f21;
+    padding: 10px 5px;
+    width: 25px;
```

Comment:
Can't we use `%` instead of `px`?
Applies to other places as well.

Reply:
This is for a fixed width toolbar, that's why px is been used.